File: src/main/java/io/github/hotlava03/baclava4j/commands/fun/FunCategory.java
package io.github.hotlava03.baclava4j.commands.fun;

import java.util.Arrays;
import java.util.List;

import io.github.hotlava03.baclava4j.commands.Category;
import io.github.hotlava03.baclava4j.commands.CategoryType;
import io.github.hotlava03.baclava4j.commands.Command;

public class FunCategory implements Category {

    @Override
    public CategoryType getType() {
        return CategoryType.FUN;
    }

    @Override
    public List<Command> getCommands() {
        return Arrays.asList(
            
        );
    }

}

File: src/main/java/io/github/hotlava03/baclava4j/commands/fun/FunCategory.kt
package io.github.hotlava03.baclava4j.commands.`fun`

import io.github.hotlava03.baclava4j.commands.Category
import io.github.hotlava03.baclava4j.commands.CategoryType
import io.github.hotlava03.baclava4j.commands.Command

class FunCategory : Category {
    override val type: CategoryType
        get() = CategoryType.FUN

    override val commands: List<Command>
        get() = listOf()
}